build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# rtl8721csm_download.sh
+
+if [ -n "$2" ];then
```

Comment:
Is there any usages of $1? Why do you use $2 without $1?
Could you add the message to say how to use this script?

Reply:
Hi @sunghan-chang ,

The $1 is currently represents the selected download option. 
![image](https://user-images.githubusercontent.com/41885176/88618452-c4fbdb80-d0cb-11ea-9e38-9a8ce4ae7cf5.png)

The $2 is an optional input. Without providing it, the serial port is set as ttyUSB1 by default.
This parameter is added only for leaving an configurable option for users. 

To download image, you can use following command:
1)  ./dbuild menu  (will provide default serial port setting)
or
2)  ./dbuild.sh download ALL ttyUSB1

Thanks!

Reply:
In this script, there is no `$1` handling. It means `$1` is redundant.
If I give the command as `./dbuild.sh download os ttyUSB1`. Does it download os binary only with bootloader?
It looks no.

Reply:
Yes, you are right!
This download function only support 'download ALL' function for now.
$1 is currently reserved and left for further use of specialized download options.

Reply:
To prevent confusion, I will add some lines to restrict the input that only 'ALL' option can be selected for now. For the not provided inputs, an reminder message will be displayed. Thanks!

Reply:
That's good.